scrc/dataset_creation/dataset_creator.py

```diff
@@ -614,6 +617,7 @@ def filter_column(row, section_attr):
             df.year = df.year.astype(int)  # convert from float to nicer int
         df.decision_id = df.decision_id.astype(str)  # convert from uuid to str so it can be saved
 
+        df = df.loc[df['facts_num_tokens_bert'] > 10]  # remove entries with less than 10 tokens
```

Comment:
Did you check that this is the only place where we filter by sequence length?

Reply:
I checked it now and as far as I understand there is a filter in the `clean_dataset` method. I tried it out but it didn't work for the judgment dataset. I couldn't find out why in a reasonable time frame, that's why I keep the `df.loc` - at least for now.

Reply:
ok, yes this method needs to be refactored. I can do that. I will also replace the magic number with a constant.